Geometry/TrackerGeometryBuilder/interface/phase1PixelTopology.h

```diff
@@ -20,6 +20,15 @@ namespace phase1PixelTopology {
 
   constexpr uint32_t numPixsInModule = uint32_t(numRowsInModule)* uint32_t(numColsInModule);
 
+  constexpr uint32_t numberOfModules = 1856;
+  
+  constexpr uint32_t layerStart[11] = {0,96,320,672,1184,1296,1408,1520,1632,1744,1856};
+  constexpr char const * layerName[10] = {"BL1","BL2","BL3","BL4",
+   		  	                  "E+1", "E+2", "E+3",
+			                  "E-1", "E-2", "E-3"
+                                          };
```

Comment:
Just to note that this introduces yet another convention for naming pixel layers. It seems to be only for debug prints, so I don't object.

Reply:
@makortel what are the "usual" pixel layer names, and where are they defined ?

Reply:
We (=CMS) have many conventions in different places

* Seeding layers are specified with `BPixN` and `FPixN_pos`/`FPixN_neg` 
https://github.com/cms-patatrack/cmssw/blob/96559f3f9ce8d05cae0996949fe379d758108ea6/RecoTracker/TkSeedingLayers/python/PixelLayerTriplets_cfi.py#L6-L11
* `PixelSubdetector` uses `enum`s `PixelBarrel` and `PixelForward` https://github.com/cms-patatrack/cmssw/blob/96559f3f9ce8d05cae0996949fe379d758108ea6/DataFormats/SiPixelDetId/interface/PixelSubdetector.h#L11
* `GeomDetEnumerators` uses `enum`s `PixelBarrel`, `PixelEndcap`, `P1PXB`, `P1PXEC`, `P2PXB`, `P2PXEC` https://github.com/cms-patatrack/cmssw/blob/96559f3f9ce8d05cae0996949fe379d758108ea6/Geometry/CommonDetUnit/interface/GeomDetEnumerators.h#L11
* `TrackerTopology` uses `pxb` and `pxf` in function prefixes https://github.com/cms-patatrack/cmssw/blob/96559f3f9ce8d05cae0996949fe379d758108ea6/DataFormats/TrackerCommon/interface/TrackerTopology.h#L160-L163
* `TrackingNtuple` python library uses `BPixN` and `FPixN+`/`FPixN-` in printouts
https://github.com/cms-patatrack/cmssw/blob/96559f3f9ce8d05cae0996949fe379d758108ea6/Validation/RecoTrack/python/plotting/ntupleDataFormat.py#L100-L112

I don't think none of these is authoritative enough to suggest a change in here (since they're for printouts only, for configuration input's I'd probably suggest the seeding layers' convention). Anyway these have the nice property (on purpose?) that the length of BPix and FPix strings are the same (that none of the other conventions have).